FlaskApp/api/routes.py

```diff
@@ -64,26 +65,29 @@ def post(self):
 
         """check if email is already registered"""
 
-        if User.return_user(data['email'].strip().lower()):
-            response = jsonify({'message': 'Email already registered'})
-            response.status_code =400
-            return response
+        check_query = 'SELECT FROM users WHERE "email" =\'{}\'and "username"=\'{}\''.format(data['email'], data['username'])
+    
+        user_exist =  return_user(con, check_query)
+      
+        
 
-        if User.return_user(data['username'].strip()):
+        if user_exist:
             response = jsonify({'message': 'username already registered'})
             response.status_code =400
             return response
 
-        new_user = User(data['name'].strip(), data['username'].strip(), data['email'].strip().lower(),generate_password_hash(data['password'].strip())).register_user()
+        new_user_query = 'INSERT INTO users (username, email, password)\
+         VALUES(\'%s\',\'%s\',\'%s\');' %(data['username'].strip(),data['email'].strip().lower(),\
```

Comment:
SQL injection possible here.
Have a look at this :arrow_right: https://security.openstack.org/guidelines/dg_parameterize-database-queries.html

Reply:
Thanks for  the feedback realy helpful